src/main/java/kr/pickple/back/address/service/AddressService.java

```diff
@@ -0,0 +1,55 @@
+package kr.pickple.back.address.service;
+
+import static kr.pickple.back.address.exception.AddressExceptionCode.*;
+
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import kr.pickple.back.address.domain.AddressDepth1;
+import kr.pickple.back.address.domain.AddressDepth2;
+import kr.pickple.back.address.dto.response.AllAddressResponse;
+import kr.pickple.back.address.dto.response.MainAddressResponse;
+import kr.pickple.back.address.exception.AddressException;
+import kr.pickple.back.address.repository.AddressDepth1Repository;
+import kr.pickple.back.address.repository.AddressDepth2Repository;
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class AddressService {
+
+    private final AddressDepth1Repository addressDepth1Repository;
+    private final AddressDepth2Repository addressDepth2Repository;
+
+    public AllAddressResponse findAllAddress() {
+        AddressDepth1 addressDepth1 = addressDepth1Repository.findAll()
```

Comment:
`p1`

```suggestion
      final AddressDepth1 addressDepth1 = addressDepth1Repository.findAll()
```

해당 클래스 내에 있는 변수들 모두 변경되는 부분들은 없는 것 같은데 final로 선언해주셔도 될 것 같습니다!

Reply:
반영 : [3f26347](https://github.com/Java-and-Script/pickple-back/pull/29/commits/3f26347b7e8756030d4882afd14a31e78819b991)
- 감사합니다~